Components/Core/Conditions/Categories/Location.lua

```diff
@@ -172,8 +172,39 @@ ConditionCategory:RegisterCondition(1,	 "INSTANCE2", {
 	end,
 })
 
+ConditionCategory:RegisterCondition(1.1,  "MYTHICKEYSTONE", {
```

Comment:
I'd prefer `KEYSTONELEVEL` as the identifier. Its more descriptive of what's being checked - MYTHICKEYSTONE could refer to anything about a M+ keystone.